NEWS

```diff
@@ -57,7 +57,39 @@ included in the vX.Y.Z section and be denoted as:
 --------------------
 
 Bug fixes/minor improvements:
+
 - Remove IB XRC support from the OpenIB BTL due to lack of support.
+- Fix a problem with MPI_IALLTOALLW when using zero-length messages.
```

Comment:
I'll fix that wording.

@artpol84 could you test 2.0.0 release to see if MLNX sees the XRC issue there as well?